azure-pipelines-analyzer-release.yml

```
@@ -0,0 +1,148 @@
+    Pattern: '*DurableTask.Analyzers.dll'
+    signConfigType: 'inlineSignParams'
+    inlineOperation: |
+      [    
+          {
+            "KeyCode": "CP-230012",
+            "OperationCode": "SigntoolSign",
+            "Parameters": {
+              "OpusName": "Microsoft",
+              "OpusInfo": "http://www.microsoft.com",
+              "FileDigest": "/fd \"SHA256\"",
+              "PageHash": "/NPH",
+              "TimeStamp": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+            },
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          },
+          {
+            "KeyCode": "CP-230012",
+            "OperationCode": "SigntoolVerify",
+            "Parameters": {},
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          }
+      ]
+    SessionTimeout: '60'
+    MaxConcurrency: '50'
+    MaxRetryAttempts: '5'
+
+# SBOM generator task for additional supply chain protection
+- task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+  displayName: 'Manifest Generator '
+  inputs:
+    BuildDropPath: '$(System.DefaultWorkingDirectory)'
+
+# dotnet pack
+# Packaging needs to be a separate step from build.
+# This will automatically pick up the signed DLLs.
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet pack WebJobs.Extensions.DurableTask.Analyzers.csproj'
+  inputs:
+    command: pack
+    packagesToPack: 'src/**/WebJobs.Extensions.DurableTask.Analyzers.csproj'
+    configuration: Release
+    packDirectory: 'azure-functions-durable-extension'
+    nobuild: true
+
+# Remove redundant symbol package(s)
+- script: |
+   echo *** Searching for .symbols.nupkg files to delete...
+   dir /s /b *.symbols.nupkg
+   
+   echo *** Deleting .symbols.nupkg files...
+   del /S /Q *.symbols.nupkg
+   
+   echo *** Listing remaining packages
+   dir /s /b *.nupkg
+  displayName: 'Remove Redundant Symbols Package(s)'
+  continueOnError: true
+
+# Digitally sign all the nuget packages with the Microsoft certificate.
+# This appears to be an in-place signing job, which is convenient.
+- task: EsrpCodeSigning@1
+  inputs:
+    ConnectedServiceName: 'ESRP Service'
+    FolderPath: '$(System.DefaultWorkingDirectory)'
+    Pattern: '*.nupkg'
+    signConfigType: 'inlineSignParams'
+    inlineOperation: |
+      [    
+          {
+            "KeyCode": "CP-401405",
+            "OperationCode": "NuGetSign",
+            "Parameters": {},
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          },
+          {
+            "KeyCode": "CP-401405",
+            "OperationCode": "NuGetVerify",
+            "Parameters": {},
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          }
+      ]
+    SessionTimeout: '60'
+    MaxConcurrency: '50'
+    MaxRetryAttempts: '5'
+
+# Make the nuget packages available for download in the ADO portal UI
+- publish: $(build.artifactStagingDirectory)
```

I noticed your pipeline completed, but no artifacts were uploaded. From the `dotnet pack` command above, it seems the expected location for the nuget package is `$(build.artifactStagingDirectory)/azure-functions-durable-extension` (see line 100). However, I also noticed that in your latest commit, you removed the `publish` task that referenced this location. The current location of `$(build.artifactStagingDirectory)` doesn't seem to be correct. Should it instead by the following?

```suggestion
- publish: $(build.artifactStagingDirectory)/azure-functions-durable-extension
```